Add bio attribute to artist resource

aonomike opened the pull request:
### What does this PR do?

Add `bio` attribute to `Artist` schema



#### Description of Task to be completed?

Add a required field `bio` to the Artist schema


#### How should this be manually tested?

1. On terminal, run command ...

`mix test`
`mix phx.server`


#### What are the relevant Github Issues ?
Fixes #55

aonomike committed:
Add about attribute to artist resource

Co-authored-by: okothkongo <34419890+okothkongo1@users.noreply.github.com>

aonomike committed:
Resolve failing controller tests

mattwr18 committed:
Merge branch 'develop' into 55_add-about-attribute-to-artist

mattwr18 commented:
@FedericoEsparza 
@aonomike and I were talking and wondering about the naming of the field, and whether it should be `about` or `bio`. Maybe it's not important and shouldn't block this from being merged, but before approving the changes we thought we check with the `Product Owner` :stuck_out_tongue_closed_eyes: :

FedericoEsparza reviewed (commented):
@mattwr18 @aonomike Yes, I like `bio` better, because it correlates with the frontend component, more closely. Please modify, accordingly.

aonomike committed:
Merge branch 'develop' into 55_add-about-attribute-to-artist

aonomike committed:
Merge branch 'develop' of github.com:AgileVentures/sing_for_needs into 55_add-about-attribute-to-artist

aonomike committed:
Update about to bio

- Update field about to bio
- Update test alias on mix to drop database
- Update seed file with artist bio

aonomike committed:
Merge branch '55_add-about-attribute-to-artist' of github.com:aonomike/sing_for_needs into 55_add-about-attribute-to-artist
